index.html

```diff
@@ -0,0 +1,141 @@
+      <p>This document specifies a set of technologies that allow a Web developer to customize the title bar of an installed Web application running in a desktop environment. These technologies are known as Window Controls Overlay (<abbr title="Window Controls Overlay">WCO</abbr>).</p>
+    </section>
+    <section id="sotd">
+      <p>This draft is meant to serve as a starting point for the standardization process of WCO and the technologies described in it might be migrated to other specifications on different working groups.</p>
+    </section>
+    <section class="informative">
+      <h2>Introduction</h2>
+      <p>Installed web apps hosted within a user agent (UA) frame are able to declare which browser display mode best meets the needs of the application via the [=manifest=] file's `display` member. Developers targeting non-mobile devices looking to create an immersive, native-like title bar for their installed application may want to utilize this area, but it is not supported via the aforementioned `display` member. Instead, the client area begins immediately below the reserved title bar area, which can create a cramped application space especially on portable devices with smaller screens.</p>
+      <p>Window Controls Overlay addresses this challenge by introducing a set of technologies throughout JS and CSS that allow to the web content area to expand over the title bar area, remaining only the area corresponding to the overlay that houses the controls of the window.</p>
+
+    </section>
+    <section>
+      <h2>Extensions to the `Navigator` interface</h2>
+      <p>
+        The [[HTML]] specification defines the <dfn>Navigator</dfn> interface, which this
+        specification extends:
+      </p>
+      <pre class="idl">
+        [SecureContext, Exposed=(Window)]
+        partial interface Navigator {
+          [SameObject] readonly attribute WindowControlsOverlay windowControlsOverlay;
+        };
+      </pre>
+      <section>
+        <h3>`windowControlsOverlay` attribute</h3>
+      </section>
+      <p>The {{navigator.windowControlsOverlay}} attribute is an instance of
+        {{WindowControlsOverlay}}. This attribute provides the current visibility of the overlay, the bounding rectangular area of the overlay, and whether there was an ongeometrychange event to that area of the overlay.
+        The <a>user agent</a> MUST run the <a>update the overlay area
+        information</a> algorithm steps to initialize the values and return
+        the {{Navigator.windowControlsOverlay}} attribute.</p>
+    </section>
+    <section data-dfn-for='WindowControlsOverlay' data-link-for=
+    'WindowControlsOverlay'>
+      <h2>
+        <dfn>WindowControlsOverlay</dfn> interface
+      </h2>
+      <pre class='idl' data-cite="DOM HTML">
+        [Exposed=Navigator]
+        interface WindowControlsOverlay : EventTarget {
+          readonly attribute boolean visible;
+          readonly attribute DomRect getBoundingClientRect;
```

Comment:
Interestingly, this doesn't match the implementation [here](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/frame/window_controls_overlay.idl?q=windowcontrolsoverlay%20-path%3Dout%20path:third_party%2Fblink). Do you know why they're different? 

Do note that we currently are accessing these differently. Devs would grab each of these as:
```
navigator.windowControlsOverlay.visible
navigator.windowControlsOverlay.getBoundingClientRect()
```